devolving_music/models/song_submission.py

```diff
@@ -31,7 +50,61 @@ def voteable(self) -> bool:
     def get_voteable_submissions(Event):
         voteable_submissions = [
                 sub
-                for sub in SongSubmission.objects.filter(event__exact=Event).order_by('?')
+                for sub in SongSubmission.objects.filter(event__exact=Event).order_by('id')
                 if sub.voteable()
             ]
         return voteable_submissions
+
+    @staticmethod
+    def check_compare(comparison_submission:"SongComparison",\
+        song1:"SongSubmission",song2:"SongSubmission"):
+        return not song1.compare_present(comparison_submission) and \
+        not song2.compare_present(comparison_submission)
+
+    @staticmethod
+    def elo_song_rating(comparison_submission:"SongComparison",song1,song2,score_range=30):
```

Comment:
Method names should describe what you're actually doing. I might suggest `update_elo_scores` here.

Reply:
you're getting the elo_song_rating not updating them